src/app/ubs/ubs-admin/components/shared/components/dialog-tariff/dialog-tariff.component.html

```diff
@@ -1,6 +1,3 @@
-<div class="d-flex english-row">
-  <div *ngIf="!deactivatePopup" class="english-hint">{{ 'ubs-tariffs-add-location-pop-up.english-hint' | translate }}</div>
```

Comment:
You should not delete it, but pass false or true to deactivatePopup from your needed component
We use the `dialog-tariff` component in several places, so your solving will create a lot of other bugs